elevation_server:
  ros__parameters:
    max_range: 0.3
    map_origin:
      x: -0.6096
      y: -0.381
    map_resolution: 0.001
    elevation_scale: 1.0
